### ISSUE TITLE
Rec model error with numeric predictor IDs

### ISSUE THREAD

**Issue by emlys** [natcap/invest#1278]
[A user reported this error](https://community.naturalcapitalproject.org/t/recreation-error/3644):
```
2023-04-17 11:04:11,853 (natcap.invest.utils) utils.prepare_workspace(166) ERROR Exception while executing natcap.invest.recreation.recmodel_client
Traceback (most recent call last):
  File "natcap\invest\utils.py", line 164, in prepare_workspace
  File "invest\cli.py", line 486, in main
  File "natcap\invest\recreation\recmodel_client.py", line 318, in execute
  File "natcap\invest\recreation\recmodel_client.py", line 1435, in _validate_same_id_lengths
TypeError: object of type 'int' has no len()
```
When the values in the predictor ID column are all numbers, they become `int`s instead of `str`s. The model should allow numeric IDs and just treat them as strings.

**Comment by emlys** [comment]
I think it would make the most sense to solve this with what's proposed in #1319 . Putting this on hold until that's implemented.

**Comment by davemfish** [comment]
> I think it would make the most sense to solve this with what's proposed in #1319 . Putting this on hold until that's implemented.

I agree that's a good solution to the problem.

A user that provides integers as their predictor ids will have a much harder time interpreting their results than if they had used meaningful strings. Maybe we should be more clear about the purpose of that column:

>id ([text](http://releases.naturalcapitalproject.org/invest-userguide/latest/en/input_types.html#text), required): A unique ~~identifier~~ name for the predictor (10 characters or less).

In hindsight "id" might be better as "name", but that would be a breaking change.

**Comment by emlys** [comment]
This was fixed in #1345. The `id` column will now be parsed as strings, whether or not it contains numeric values.
